verification/curator-service/ui/src/components/App.tsx

```diff
@@ -86,38 +95,43 @@ class App extends React.Component<{}, User> {
     }
 }
 
-function Linelist(): JSX.Element {
-    return <LinelistTable />;
-}
-
-function Sources(): JSX.Element {
-    return <SourceTable />;
-}
-
 interface HomeProps {
     user: User;
 }
 class Home extends React.Component<HomeProps, {}> {
+    hasAnyRole(requiredRoles: string[]) {
+        return this.props.user.roles?.some((r: string) =>
+            requiredRoles.includes(r),
+        );
+    }
     render(): JSX.Element {
         return (
-            <nav>
-                <Link to="/cases">Linelist</Link>
-                <br />
-                <Link to="/sources">Sources</Link>
-                <br />
+            <div>
+                {!this.props.user.email && <div>Login to access Epid</div>}
```

Comment:
nit, up to you but you can use the ternary operator here instead of repeating the check
{!email ? login please : show stuff}

Reply:
Good call - done.